### ISSUE TITLE
MITM

### ISSUE THREAD

**Issue by evertking** [cSploit/android#734]
Will this ever work again?  As you all know. It's gives cert missing errors.. warning.. your being hacked! Mitm is pretty much useless and it's not any fault to the devs, I'm just wondering.

**Comment by IwraStudios** [comment]
i can assure you: Yes. the reason why it doesn't work anymore/yet is because of a few things:

1. Development for cSploit 2 is in progress(this will have the MITM fully working if everything goes according to the plan).
2. Updates, lots and lots of Updates: of course if security can be broken the people who supply the security try to fix it and then researchers find new holes and then it loops back around. while cSploit(1) was in development the fix wasn't implemented yet and when it was the mitm stopped working

TL;DR
We will have it fixed in cSploit 2
